pkg/apply/solver/solver_test.go

```diff
@@ -644,7 +650,7 @@ func TestTaskQueueBuilder_AppendPruneWaitTasks(t *testing.T) {
 				case *taskrunner.WaitTask:
 					actWaitTask := toWaitTask(t, actualTask)
 					assert.Equal(t, len(expTsk.Ids), len(actWaitTask.Ids))
-					if !object.SetEquals(expTsk.Ids, actWaitTask.Ids) {
+					if !object.IsSetEqual(expTsk.Ids, actWaitTask.Ids) {
```

Comment:
Remove this change. If you want to advocate for a name change, please do it in another PR.